modes/basic-test-mode/src/toolbarButtons.ts

```diff
@@ -165,13 +151,14 @@ const toolbarButtons: Button[] = [
               context: 'CORNERSTONE',
             },
           ],
-          'Ellipse Tool'
-        ),
-        ToolbarService._createToolButton(
-          'CircleROI',
-          'tool-circle',
-          'Circle',
-          [
+          evaluate: 'evaluate.cornerstoneTool',
+        }),
+        ToolbarService.createButton({
+          id: 'CircleROI',
+          icon: 'tool-circle',
+          label: 'Circle',
+          tooltip: 'Circle ROI Tool',
+          commands: [
             {
               commandName: 'setToolActive',
```

Comment:
If the set tool active took an array of tool group Ids, then you could do this all in one - and just use a single fixed commands list that includes the tool gorup ids.